lucille-examples/lucille-distributed-example/runner/Dockerfile

```diff
@@ -1,13 +1,20 @@
 FROM ubuntu
 RUN apt-get upgrade
-RUN apt-get update && apt-get install curl -y
-RUN apt-get install lsof bc openjdk-11-jre-headless -y --fix-missing
+RUN apt-get update && apt-get install curl -y && apt-get install maven -y
+RUN apt-get install lsof bc openjdk-17-jre-headless -y --fix-missing
 
 # we copy local build artifacts into the image that we're building
 COPY target/ /target/
 COPY conf/ /conf/
 COPY runner/scripts/ /scripts/
 RUN mkdir /output/
 
+# copy the test class into the container
+COPY runner/pom.xml /pom.xml
+COPY /src/test/java/com/kmwllc/lucille/distributed/DistributedIT.java /src/test/java/com/kmwllc/lucille/distributed/DistributedIT.java
```

Comment:
If we've already run a build and generated the target directory, wouldn't the compiled DistributedIt.class file be in there? Is there a way to use that already compiled class without having to copy the java source inside the container and build it there a second time?

Reply:
yes ur right. Copied the target folder to the container instead of the java class itself in [674abf2](https://github.com/kmwtechnology/lucille/pull/206/commits/674abf24013a54cbcf1464674f45f449eb49f6bc)